src/hotspot/share/gc/g1/g1_globals.hpp

```diff
@@ -277,6 +277,12 @@
           "as a percentage of the heap size.")                              \
           range(0, 100)                                                     \
                                                                             \
+  product(uint, G1OldCSetGroupSize, 5, EXPERIMENTAL,         \
+          "The maximum number of old CSet regions in a collection group. "  \
+          "These will be evacuated in the same GC pause. The first group "  \
+          "may exceed this limit depending on G1MixedGCCountTarget.")       \
```

Comment:
Maybe this is better, not sure. We should explain why the "first group" is special.

```suggestion
  product(uint, G1OldCSetGroupSize, 5, EXPERIMENTAL,         \
          "The maximum number of old CSet regions in a collection group. "  \
          "All regions in a group will be evacuated in the same GC pause. The first group calculated after marking from marking candidates "  \
          "may exceed this limit as it is calculated based on G1MixedGCCountTarget.")       \
```